includes/sanitizers/class-amp-link-sanitizer.php

```diff
@@ -31,14 +31,32 @@ class AMP_Link_Sanitizer extends AMP_Base_Sanitizer {
 	 */
 	const DEFAULT_META_CONTENT = 'AMP-Redirect-To; AMP.navigateTo';
 
+	/**
+	 * The rel attribute value for AMP links.
+	 *
+	 * @var string
+	 */
+	const REL_VALUE_AMP = 'amphtml';
+
+	/**
+	 * The rel attribute value that will force non-AMP links.
+	 *
+	 * Normally, in a paired mode, links to the same origin will be for AMP.
+	 * But by adding this rel value, the link will be to non-AMP.
+	 *
+	 * @var string
+	 */
+	const REL_VALUE_NON_AMP_TO_AMP = 'noamphtml';
+
 	/**
 	 * Placeholder for default arguments, to be set in child classes.
 	 *
 	 * @var array
 	 */
 	protected $DEFAULT_ARGS = [ // phpcs:ignore WordPress.NamingConventions.ValidVariableName.PropertyNotSnakeCase
-		'paired'       => false, // Only set to true when in a paired mode (will be false when amp_is_canonical()). Controls whether query var is added.
-		'meta_content' => self::DEFAULT_META_CONTENT,
+		'paired'             => false, // Only set to true when in a paired mode (will be false when amp_is_canonical()). Controls whether query var is added.
+		'meta_content'       => self::DEFAULT_META_CONTENT,
+		'excluded_amp_links' => [], // URLs in this won't have AMP-to-AMP links in a paired mode.
```

Comment:
```suggestion
		'excluded_urls' => [], // URLs in this won't have AMP-to-AMP links in a paired mode.
```

and change the other instances of this string.

Reply:
Sure, that parameter is renamed in https://github.com/ampproject/amp-wp/pull/4146/commits/b6dda38531c77838a944ef1d119b635115b1bce7